src/main/java/com/esri/arcgisruntime/toolkit/FeatureTemplatePicker.java

```diff
@@ -16,341 +16,221 @@
   /**
-   * Property controlling the height of the template symbols. The height is specified in device independent pixels.
+   * Property containing the list of feature layers displayed. The order of display matches the list order.
    *
    * @return the property
-   * @since 100.6.0
-   */
-  public SimpleIntegerProperty symbolHeightProperty() {
-    return symbolHeightProperty;
-  }
-
-  /**
-   * Sets the value of the {@link #symbolHeightProperty()}.
-   *
-   * @param height the height
-   * @since 100.6.0
+   * @since 100.7.0
    */
-  public void setSymbolHeight(int height) {
-    symbolHeightProperty.set(height);
+  public ListProperty<FeatureLayer> featureLayersProperty() {
+    return featureLayers;
   }
 
   /**
-   * Gets the value of the {@link #symbolHeightProperty()}.
+   * Gets the value of the {@link #featureLayersProperty()}.
    *
-   * @return the height
-   * @since 100.6.0
+   * @return the list of feature layers
+   * @since 100.7.0
    */
-  public int getSymbolHeight() {
-    return symbolHeightProperty.get();
+  public ObservableList<FeatureLayer> getFeatureLayers() {
+    return this.featureLayers.get();
   }
 
   /**
-   * A property controlling if the template names should be shown. Set this to true to display the template names, false
-   * otherwise.
+   * Sets the value of the {@link #featureLayersProperty()}.
    *
-   * @return the property
-   * @since 100.6.0
+   * @param featureLayers the list of feature layers
+   * @since 100.7.0
    */
-  public SimpleBooleanProperty showTemplateNamesProperty() {
-    return showTemplateNamesProperty;
+  public void setFeatureLayers(ObservableList<FeatureLayer> featureLayers) {
+    this.featureLayers.set(featureLayers);
   }
 
   /**
-   * Sets the value of the {@link #showTemplateNamesProperty()}.
+   * Gets the list of feature template groups in the picker.
    *
-   * @param show true to show template names, false otherwise
-   * @since 100.6.0
+   * @return feature template groups
    */
-  public void setShowTemplateNames(boolean show) {
-    showTemplateNamesProperty.set(show);
+  public ObservableList<FeatureTemplateGroup> getFeatureTemplateGroups() {
+    return featureTemplateGroups.get();
   }
 
   /**
-   * Gets the value of the {@link #showTemplateNamesProperty()}.
+   * Read only list of feature template groups in the picker.
    *
-   * @return true if template names are shown, false otherwise
-   * @since 100.6.0
+   * @return read-only feature template groups list property
+   * @since 100.7.0
    */
-  public boolean isShowTemplateNames() {
-    return showTemplateNamesProperty.get();
+  public ReadOnlyListWrapper<FeatureTemplateGroup> featureTemplateGroupsProperty() {
+    return featureTemplateGroups;
   }
 
   /**
-   * Property controlling if the feature layer names are shown above that feature layer's templates. Set this to true to
-   * display the feature layer names, false otherwise.
+   * Gets the orientation of this control. See skin for effect.
```

Comment:
I imagine that would depend on the skin. An accordion type skin would probably ignore orientation. This goes back to my earlier question about skin-agnostic doc: https://github.com/Esri/arcgis-runtime-toolkit-java/pull/32#discussion_r391229918.

Reply:
How about we document the current skin behaviour on the assumption that is the skin the majority of people will be using? It could say something like "with the default skin orientation does...".